### PR TITLE
split the original training data into a new training set and a new test set

### PR DESCRIPTION
**Pull request by DimPolissiou**

### CODE REVIEW

File: Dimitris/split.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import os.path as path
+import pandas as pd
+from pandas import DataFrame
+from sklearn.model_selection import train_test_split
+
+train_data_path = path.join(path.pardir, 'train.csv')
+data = pd.read_csv(train_data_path)
+new_test: DataFrame
```
Comment (DimPolissiou) [inline Dimitris/split.py]: I only added this to help intellisense and forgot to delete it, but this is such a throwaway script that I'm not even going to amend it.
